jobs/warcorrespondent.md

```diff
@@ -0,0 +1,28 @@
+## Abstract
+
+War Correspondent is a journalist role, able to pick at the beginning of the round whether it's marine aligned or independent (The only difference being one can take orders from an XO and gets a rank, with the other not having a military rank). It's meant to provide more roleplay opportunities and add more flavour. It provides more opportunities because it's contextually different from CL who is usually maligned by Marine players, with this role it'll be completely different as WC starts with 0 W-Y affiliation and probably won't get such derision, along with just being a contextually different role with different objectives than the CL. 
+
+## Goals
+
+1. Create a role that's interesting and provides new oppirtunities along with spicing up the game, the WC is contextually different from any other role in the game, providing leeway for players to get creative with the way they play their character in this role.
+2. Create a roleplay-centric role that's different enough from the CLto be its own thing, 
+
+## Non-goals
+
+1. Making another CL, the CL has a specific role as the W-Y guy. This role should not just be just that, it's contextually different and provides players a choice with who they align with.
+2. Shifting the balance of the game, WC has no impact on the general balance of the game, and shouldn't. It gets the same gear as a PFC would.
+3. Giving new opportunities for grief
+
+## Content
+
+Roleplay role similar to CL but gets their own vendor and armor for deployment, similar to PFC in that way but is contextually different and not provided enough to be a complete match for PFC, leading to more roleplay and less PFC'ing. It's expected that people will get creative like with CL and try to make entertaining roleplay scenarios, whether it be interviewing people or getting together group photos.
+
+## Alternatives
+
+This is such a specific role niche that thinking of an alternative is very difficult, Head dev specifically requested the ability to choose alignment at roundstart as such removing that feature isn't possible. The only other suggested change was to remove their weaponry, which wouldn't do much of anything other than forcing them to head to req at roundstart.
+
+## Potential Changes
+Gear may need to be tweaked to be appropriate for the role. 
+Camera might need to be tweaked to be more unique from other variations.
+Sprites could be changed depending on feedback from reviewers.
+Role may face issues that CL faces like people complaining about lack of an incentive to do things.
```

Comment:
Remove all of these except the bottom part and explain what change you'd make to fix this last problem